packages/yourloops/pages/login/login.tsx

```diff
@@ -43,27 +43,50 @@ import Grid from "@material-ui/core/Grid";
 import IconButton from "@material-ui/core/IconButton";
 import InputAdornment from "@material-ui/core/InputAdornment";
 import TextField from "@material-ui/core/TextField";
+import Link from "@material-ui/core/Link";
 
 import Visibility from "@material-ui/icons/Visibility";
 import VisibilityOff from "@material-ui/icons/VisibilityOff";
 
 import brandingLogo from "branding/logo.png";
 import { useAuth } from "../../lib/auth/hook/use-auth";
+import LoginFooterLink from "./login-footer-link";
 
-const loginStyle = makeStyles((/* theme: Theme */) => {
+const loginStyle = makeStyles((theme: Theme) => {
   return {
     mainContainer: { margin: "auto" },
+    root: { minHeight: "100vh" },
     loginButton: {
       marginLeft: "auto !important",
     },
+    rightLink: {
+      padding: theme.spacing(0.5),
+      textAlign: "start",
+      fontSize: "small",
+    },
+    centeredLink: {
+      padding: theme.spacing(0.5),
+      textAlign: "center",
+      color: "#109182",
+    },
+    leftLink: {
+      padding: theme.spacing(0.5),
+      textAlign: "end",
+      fontSize: "small",
+    },
+    selection: {
+      padding: theme.spacing(2),
+      textAlign: "center",
+      fontSize: "small",
+    },
   };
 });
 
 /**
  * Login page
  */
 function Login(props: RouteComponentProps): JSX.Element {
-  const { t } = useTranslation("yourloops");
+  const { t } = useTranslation();
```

Comment:
I would have prefer we use the new translation file.
But it need to be discuss with marine maybe.